basyx.aasrepository/basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/TestAasRepositoryHTTP.java

```diff
@@ -38,7 +38,7 @@
  * {@link AasRepository} HTTP/REST API
  * 
  * @author schnicke, danish
- *
+ * 
```

Comment:
Hi Gerhard, can you remove that whitespace please. I will merge this afterwards.

Btw, you can trigger the CI using empty commits: git commit --allow-empty -m "Trigger Build"

Reply:
Actually, this whitespace makes sense